src/parser/jobs/ast/modules/Defensives.tsx

```diff
@@ -12,5 +12,7 @@ export class Defensives extends CoreDefensives {
 		this.data.actions.HOROSCOPE,
 		this.data.actions.NEUTRAL_SECT,
 		this.data.actions.COLLECTIVE_UNCONSCIOUS,
+		this.data.actions.LIGHTSPEED,
+		this.data.actions.SWIFTCAST,
```

Comment:
It _feels_ weird to me to call either of these "Defensives" but I get why they're here. It might make more sense long term to have a separate "utility" cooldowns report, or update the description/titling on Defensives to account for both, but I'm not going to hold up this PR for that :)

Reply:
That's fair -- would be good to have a utility module in a similar sense
I was loosely thinking they are "defensives" in a way that keeps ABC for healing or raises